.github/workflows/test-command.yml

```diff
@@ -64,6 +64,9 @@ jobs:
       # Beside PyEnv, this does not set any runtimes up because it uses an AMI image that has everything pre-installed. See https://github.com/airbytehq/airbyte/issues/4559/
       - name: Install Pyenv
         run: python3 -m pip install virtualenv==16.7.9 --user
+      - uses: actions/setup-java@v1
```

Comment:
I'm not sure if the underlying AMI this is using has Java, we should run a test with a Java connector just in case.

Reply:
ah noticed you got this working with the redshift destination, nice!